### DISCUSSION TITLE
Limit more than 10.

### DISCUSSION THREAD

**Discussion by knofte** [postfixadmin/postfixadmin#815] [General]
I'm trying to figure out how to show more than 10 boxes/aliases per page on list-virtual.php, however I can not find where the Limit is specified. I've foudnt his line.
$fDisplay = (int) safepost('limit', safeget('limit', safesession('list-virtual:limit')));
However, not really sure where these are set :)

Is it related to this in config.inc.php perhaps?
$CONF['page_size'] = '10'; 

I was looking/grepping for 'limit', but perhaps this is correct setting? :)

**Comment by bnchdan** [comment]
HI,
The limit per page is set here 
https://github.com/postfixadmin/postfixadmin/blob/279e7f682d734915e4491b7532ae2d6bb36f1cd4/public/list-virtual.php#L191
where https://github.com/postfixadmin/postfixadmin/blob/279e7f682d734915e4491b7532ae2d6bb36f1cd4/public/list-virtual.php#L41


you can modify $page_size or define a new $CONF element in the config file.  I don't know if the $CONF['page_size'] is also used for something else

**Comment by knofte** [comment]
Thanks for the quick response @bnchdan . Appreciated.

I've changed the page_size to 15 and it looks good. Even if it's used for something else, I doubt anything critical would break :)
